compiler/test/dotty/tools/dotc/BootstrappedOnlyCompilationTests.scala

```diff
@@ -28,6 +28,8 @@ class BootstrappedOnlyCompilationTests {
       compileFilesInDir("tests/pos-macros", defaultOptions),
       compileFilesInDir("tests/pos-custom-args/semanticdb", defaultOptions.and("-Xsemanticdb")),
       compileDir("tests/pos-special/i7592", defaultOptions.and("-Yretain-trees")),
+      compileDir("tests/pos-special/i11251", defaultOptions.and("-Ycheck:macros")),
```

Comment:
You should be able to out these tests in `tests/pos-macros`. `Ycheck` is already enabled there.

Reply:
Hmm, interesting,  when I move i11331. to pos-macros. -- tests passed.  when moving to pos-special and compile this dir i11331 in separate steps - the test failed. 
(the result of the check depends on the number of files in the compilation?)

Reply:
moved i11251 to pos-macros